app/views/user/users/show.html.erb

```diff
@@ -4,4 +4,6 @@
 <p><%= current_user.city %>, <%= current_user.state %></p>
 <p><%= current_user.zip %></p>
 
-<%= link_to "Edit My Profile or Password", profile_edit_path %>
\ No newline at end of file
+<%= link_to "Edit My Profile or Password", profile_edit_path %>
+
+<%= link_to "My Orders", user_orders_path %>
```

Comment:
After re-reading the user story, looks like we need a condition so that the user sees this link only if they have orders.

Reply:
As requested, I added a conditional statement in the view, so that the user only sees the "My Orders" link if they have placed orders.